.github/workflows/ci.yml

```diff
@@ -26,6 +49,14 @@ jobs:
     # For pull requests it's not necessary to checkout the code
     - uses: "actions/checkout@v4"
 
+    - name: "Setup upterm session"
+      if: ${{ (true == inputs.enableUpterm) && (inputs.breakOnJob == github.job) }}
+      uses: owenthereal/action-upterm@v1
+      with:
+        limit-access-to-actor: true
+        limit-access-to-users: ${{ inputs.allowedUptermUsers }}
+        wait-timeout-minutes: 15
+
```

Comment:
Could these duplicated blocks go in `.github/actions/common-setup`?

Maybe at the expense of not being able to debug some jobs that don't currently use common but are less likely to need debugging.

Reply:
I don't believe so, no.
Since these really are steps that run within that job.
I don't know of a way to "template" job with GitHub actions.

Reply:
I appreciate having this, but yeah all these duplications become a bit messy :D